src/mutations/item.test.ts

```diff
@@ -1041,4 +1043,46 @@ describe('Items Mutations', () => {
       });
     });
   });
+
+  describe('useDeleteItemThumbnail', () => {
+    const mutation = mutations.useDeleteItemThumbnail;
+    const items = generateFolders(1);
+
+    const itemId = items[0].id;
+    it('Delete item thumbnail', async () => {
+      const route = `/${buildDeleteItemThumbnailRoute(itemId)}`;
+
+      queryClient.setQueryData(itemKeys.single(itemId).allThumbnails, items[0]);
+
+      const response = [StatusCodes.NO_CONTENT];
+
+      const endpoints = [
+        {
+          response,
+          method: HttpMethod.Delete,
+          route,
+        },
+      ];
+
+      const mockedMutation = await mockMutation({
+        endpoints,
+        mutation,
+        wrapper,
+      });
+
+      await act(async () => {
+        mockedMutation.mutate(itemId);
+        await waitForMutation();
+      });
+      const data = queryClient.getQueryData(
+        itemKeys.single(itemId).allThumbnails,
+      );
+      expect(data).toBeTruthy();
```

Comment:
I think the check should be checking the state of the query, that the query is actually invalidated.